### PR TITLE
Add x-mirage 1.05.2

### PR DESCRIPTION
**Pull request by donbobka**
1.05.2 is a free version of x-mirage. License key can be obtained [here](https://stacksocial.com/sales/the-mac-freebie-bundle-3-0)

### CODE REVIEW

File: Casks/x-mirage152.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'x-mirage152' do
+  version '1.05.2'
+  sha256 '2c8cb1d4969ac5ff644c98c95ba1c29cd4e22a2a303331cbf2065ec20c84013a'
+
+  url "http://dl.x-mirage.com/x-mirage(#{version}).release.dmg"
+  homepage 'http://www.x-mirage.com/x-mirage/'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/x-mirage152.rb]: Please change to `license :commercial`.

File: Casks/x-mirage152.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'x-mirage152' do
```
Comment (vitorgalvao) [inline Casks/x-mirage152.rb]: Please change to `x-mirage1052` (don’t forget the cask name).

**Comment by vitorgalvao** [comment]
The version on that website is advertised as `1.02.9`, not `1.05.2`. In addition, this doesn’t appear to be free at all, just a temporary promotion.

**Comment by donbobka** [comment]
Fixed

**Comment by donbobka** [comment]
About free version and promotion:
Yup it's a promotion, but that's promotion available at least from April, 2014. Unfortunately, license key obtained in this way accepted only up to 1.05.2.

Answer from X-Mirage support about upgrading my license key for latest version(1.06.6)
> Hi,
>
> Thanks for your feedback.
>
> Free versions can't enjoy free major update.
> 
> You need to purchase a license if you want to use the latest version.
>
> Or you can download the version you got here:
>
> http://dl.x-mirage.com/x-mirage(1.05.2).release.dmg
> 
> Should you have any question, please feel free to contact us.
> 
> Sincerely Yours,
> 
> X-Mirage Support

**Pull request merged**